inference-engine/samples/speech_sample/main.cpp

```diff
@@ -664,9 +664,38 @@ int main(int argc, char *argv[]) {
             genericPluginConfig.insert(std::begin(gnaPluginConfig), std::end(gnaPluginConfig));
         }
         auto t0 = Time::now();
+        std::vector<std::string> outputs;
         ExecutableNetwork executableNet;
-
         if (!FLAGS_m.empty()) {
+            if (!FLAGS_oname.empty()) {
```

Comment:
i think that imported model also should be under control of that option - we dont have a way to reduce or change number of outputs during model exporting yet, so the only way to fix possible outputs  order issue - to enable that options for both modes